rest-api/src/main/java/gov/cms/qpp/conversion/api/services/DbServiceImpl.java

```diff
@@ -28,10 +29,10 @@
 
 	private static final Logger API_LOG = LoggerFactory.getLogger(Constants.API_LOG);
 
-	@Autowired
+	@Inject
 	private DynamoDBMapper mapper;
 
-	@Autowired
+	@Inject
```

Comment:
I haven't really looked into constructor injection, but why can't we use it in this class?

Reply:
the base class `AnyOrderActionService` has an injected dependency as well. `@Inject` strikes me as being the cleaner alternative here rather than having a pass-through dependency to a base class constructor.